cmd/wego/version/cmd.go

```diff
@@ -22,16 +23,36 @@ var Cmd = &cobra.Command{
 }
 
 func runCmd(cmd *cobra.Command, args []string) {
-	if checkResponse, err := checkpoint.Check(&checkpoint.CheckParams{
+	logStr, err := CheckVersion()
+	if err != nil {
+		log.Error(err)
+	}
+
+	if logStr != "" {
+		log.Infof(logStr)
```

Comment:
`Infof` is usually used when arguments need to be passed in, you might want to try `log.Info`.